### PR TITLE
Small fixes to PDP Slider, adds dragger cursor illustration

### PR DESCRIPTION
**Pull request by chrismekim**
### Description

- Jake has noticed some little bugs with the slider drag in Safari (mobile and desktop), where the drag seems to get stuck and/or move slowly. It's a bit difficult to reliably reproduce these bugs, making it hard to debug. Generally, the slider seems choppier on Safari vs Chrome.
- In this PR I've added `waitForTransition: false` to see if that makes a difference for Jake. I was thinking maybe he could be dragging before the carousel has stopped moving. For me on Safari, it seems to slightly improve the choppiness.
- [Docs](https://splidejs.com/options/): `waitForTransition: If false, navigation, drag and swipe work while the slider is running.`
- This PR also adds a dragger illustration to replace the cursor. 

### Type(s) of changes

- [x] Bug fix

### Motivation for PR

- See description
### How Has This Been Tested?

https://1ujiu2aup97rx8g2-52674822324.shopifypreview.com

### CODE REVIEW

File: theme/scripts/productImagesSlider.ts [hunk 1]
```diff
@@ -16,15 +16,12 @@ export default(function() {
 
 		setup(slider: HTMLElement) {
 			new Splide(slider, {
-				classes: {
```
Comment (chrismekim) [inline theme/scripts/productImagesSlider.ts:19]: Removed because it seems like we no longer need these classes since we've set `arrows: false`

**Review by uditdesai — CHANGES_REQUESTED** [review]
Looks good. Just requesting a change in regard to the cursor style.

In the small area between images, I see the cursor flash and switch back to the regular cursor. Could we apply the style to the container instead maybe so this doesn't happen?
https://user-images.githubusercontent.com/14059589/122944773-15b09e80-d346-11eb-9d58-55213b30e0d2.mov

Also using the style tag inside the .liquid file is a great idea. Ik it's possible to also add the .liquid extension to scss files allowing them to access liquid variables. I'm not sure if this needs a special configuration or messes up our scss but it may be worth exploring?

Also, I think this did solve Jake's bug about not being able drag it again, however it is still a bit buggy on Safari compared to Chrome :(

**Comment by chrismekim** [comment]
@uditdesai moved the cursor styling to the container! Thanks for the suggestion :)

Still haven't been able to figure out why it feels choppier on Safari vs Chrome! Hopefully Jake sees some improvement when he reviews the entire PDP. If not, I'll take another whack at it, or maybe we can think about changing slider libraries. What do you think?

**Review by uditdesai — APPROVED** [review]
Cursor bug fixed!

It seems to work fine on Chrome and Safari mobile and this PR definitely fixed the bug of not being able to swipe again mid-transition so I think it's good to go. If Jake feels like it's still quite choppy, we can think about changing libraries potentially.

**Pull request merged**